packages/odyssey-react/src/components/Heading/Heading.theme.ts

```diff
@@ -27,6 +28,7 @@ export const theme: ThemeReducer = (theme) => ({
   Level4LineHeight: theme.FontLineHeightHeading4,
   Level5LineHeight: theme.FontLineHeightHeading5,
   Level6LineHeight: theme.FontLineHeightHeading6,
+  /* eslint-enable @okta/odyssey/no-invalid-theme-properties */
```

Comment:
cc @edburyenegren-okta these tokens violate our numbers should only appear as suffix rule but not gonna worry too much for now